src/components/Discipline.js

```diff
@@ -57,17 +57,17 @@ const styles = EStyleSheet.create({
     fontFamily: 'OpenSans-Semibold',
     backgroundColor: '$transparent',
     color: '$textColor',
-    fontSize: 22,
-    lineHeight: 30,
-    width: '100% - 120'
+    fontSize: 28,
+    lineHeight: 38,
+    width: '100% - 140'
```

Comment:
Yeah, I think this looks weird in RN since they don't use units, plus percents only work on the full device size, not container size.  Subtracting 140 is to get rid of the width of the Zoo icon container (80), the right arrow icon (20) and the right / left margins and padding (50).  Although this adds up to 150 now that I'm looking at it. 👎   I'll fix that and also convert these values into variables, I think having them named may help this calculated value make more sense and be less math-error-prone.